internal/pkg/aws/elbv2/elbv2.go

```diff
@@ -20,12 +21,18 @@ import (
 const (
 	// TargetHealthStateHealthy wraps the ELBV2 health status HEALTHY.
 	TargetHealthStateHealthy = elbv2.TargetHealthStateEnumHealthy
+	// LoadBalancerSchemeInternetFacing wraps the ELBV2 LB scheme "internet-facing".
+	LoadBalancerSchemeInternetFacing = elbv2.LoadBalancerSchemeEnumInternetFacing
```

Comment:
are these used anywhere (or will be used anywhere)?

Reply:
Maybe for validation eventually? But seems like the info returned from the API call doesn't need to be validated. Happy to delete.

Reply:
yeah - I will +1 to delete them until we need them 👀 !

Reply:
It's weird they passed checks. 🤔